christin/index.js

```diff
@@ -0,0 +1,182 @@
+//?Addition.Write a program to add up the numbers 1 to 20.
+
+for (let i = 0; i <= 20; i++) {
+  let sum = i + i;
+  console.log("1.", sum);
+}
+
+//0 + i = 1
+//1 + i = 3
+
+//?There are i bottles of beer on the wall.Write a program that will output, "There is one bottle of beer on the wall." "There are two bottles of beer on the wall" up until there are five bottles.
+
+for (let i = 0; i <= 5; i++) {
+  console.log(`2: There is ${i} bottle of beer on the wall.`);
+}
+
+//? 3. The odd / even reporter.Write a program that will iterate from 0 to 20. For each iteration, it will check if the current number is even or odd, and report that to the screen(e.g. "2 is even").
+
+for (i = 0; i <= 20; i++) {
+  if (i % 2 === 0) {
+    console.log(`3: ${i} is even`);
+  } else {
+    console.log(`3: ${i} is odd`);
+  }
+}
+
+//? 4. Multiplication Tables.Write a program that will iterate from 0 to 10. For each iteration of the for loop, it will multiply the number by 9 and log the result(e.g. "2 * 9 = 18").Bonus: Use a nested for loop to show the tables for every multiplier from 1 to 10(100 results total).
+
+let x;
+
+for (let i = 0; i <= 10; i++) {
+  sum = [i * 9];
+  console.log("4.", sum);
+
+  for (let i = 0; i <= 10; i++) {
+    sum2 = [sum * i];
+    console.log("4.", sum2.join(","));
+  }
+}
+
+//? Fizz Buzz Write a program which iterates the integers from 1 to 100. But for multiples of three print "Fizz" instead of the number and for the multiples of five print "Buzz". For numbers which are multiples of both three and five print "FizzBuzz".
+
+for (let i = 0; i <= 100; i++) {
+  if (i % 3 === 0 && i % 5 === 0) {
+    console.log("5. FizzBuzz");
+  } else if (i % 3 === 0) {
+    console.log("5. Fizz");
+  } else if (i % 5 === 0) {
+    console.log("5. Buzz");
+  } else {
+    console.log("5.", i);
+  }
+}
+
+//? 6. Sum of Multiples
+//Write a program to add the multiples of 3 and 5 under 1000
+
+for (let i = 0; i < 1000; i++) {
+  if (i % 3 === 0 && i % 5 === 0) {
+    let sum3 = i + i;
+    console.log("6.", sum3);
+  } else {
+  }
+}
+
+//7. Write programs that produce the following outputs:
+// 100 200 300 400 500 600 700 800 900 1000
+let arr3 = [100, 200, 300, 400, 500, 600, 700, 800, 900, 1000];
+
+for (let i = 0; i < 10; i++) {
+  console.log("7.", arr3[i]);
+}
+
+//1 2 4 8 16 32 64 128
+
+for (let i = 2; i <= 9; i++) {
+  let res = Math.pow(i, 2);
+  //break;
+  console.log("8a.", res);
+}
+
+//0 2 4 6 8 10
+for (let i = 0; i <= 5; i++) {
+  let res3 = i + i;
+  console.log("8b.", res3);
+}
+//3 6 9 12 15
+
+for (let i = 0; i <= 5; i++) {
+  let res4 = 3 * i;
+  console.log("8c.", res4);
+}
+// 9 8 7 6 5 4 3 2 1 0
+
+for (let i = 9; i >= 0; i--) {
+  let res5 = i;
+  console.log("8d.", res5);
+}
+
+// 1 1 1 2 2 2 3 3 3 4 4 4
+
+const sum7 = [];
```

Comment:
// When I run the code it prints 0
- Inside of outer for loop you wrote 1 < 4, it should be i < 4, that's why you face endless loop, therefore you can get rid of break
- In inner loop instead of m = 0 ; m < 3 we need m=1; m<5
- After that we have to push m instead of i inside of sum7
- Now if you try to print sum 7 you will get 1, 2, 3, 4, 1, 2, 3, 4, 1, 2, 3, 4 ,1 , 2 , 3 , 4

Reply:
solved, right.